catroid/src/main/java/org/catrobat/catroid/content/bricks/GoToBrick.java

```diff
@@ -305,7 +306,9 @@ public boolean onTouch(View paramView, MotionEvent paramMotionEvent) {
 
 	@Override
 	public Brick clone() {
-		return new GoToBrick(destinationSprite);
+		GoToBrick copy = new GoToBrick(destinationSprite);
+		copy.spinnerSelection = spinnerSelection;
+		return copy;
```

Comment:
probably.  I did not (yet) have time to check if this brick contains other bugs as well *let me have a look*